tutorials/using-cloud-vpn-with-alibaba-with-redundancy

```diff
@@ -0,0 +1,950 @@
+Use these settings for the procedures in the subsections that follow.
+
+<table>
+<thead>
+<tr>
+<th><strong>Setting</strong></th>
+<th><strong>Description or value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IPsec Mode</td>
+<td>ESP+Auth Tunnel mode (Site-to-Site)</td>
+</tr>
+<tr>
+<td>Auth Protocol</td>
+<td>Pre-shared Key (psk)</td>
+</tr>
+<tr>
+<td>Shared Secret</td>
+<td>Also known as an IKE pre-shared key. Choose a strong password by following
+<a
+href="https://cloud.google.com/vpn/docs/how-to/generating-pre-shared-key">these
+guidelines</a>. The shared secret is very sensitive as it allows access
+into your network.</td>
+</tr>
+<tr>
+<td>Start</td>
+<td><code>Auto</code> (on-premises device should automatically restart the
+connection if it drops)</td>
+</tr>
+<tr>
+<td>PFS (Perfect Forward Secrecy)</td>
+<td>group1, group2, group5, group14, group24</td>
+</tr>
+<tr>
+<td>IKE ciphers</td>
+<td>aes, aes192, aes256, des, 3des <br>(For details about IKE ciphers for IKEv1 or IKEv2 supported by GCP,
+including the additional ciphers for PFS, see <a
+href="https://cloud.google.com/vpn/docs/concepts/supported-ike-ciphers">Supported
+IKE Ciphers</a>).</td>
+</tr>
+</tbody>
+</table>
+
+#### Recommended Alibaba Cloud VPN Gateway IKE proposal and policy
+Below are fields you may be asked to complete for the Alibaba Cloud side configurations. 
+The defaults indicated below will work with the defaults used on the GCP side.
+
+-  **Encryption algorithm**—<code>aes</code> 
+-  **Integrity algorithm**—<code>sha1</code>
+-  **Diffie-Hellman group**—<code>group2</code>
+-  **Lifetime/SA Life Cycels**—<code>86400</code> seconds 
+
+
+## Configuration Overview
+
+The Google Cloud VPN with Alibaba Cloud VPN Gateway configuration is comprised of the following steps.
+
+1. Configure the GCP side
+1. Configure the Alibaba Cloud side
+
+## Configure the GCP side
+
+This section covers the steps for creating a redundant GCP IPsec VPN using static routing.
+Both route-based Cloud VPN and policy-based Cloud VPN use static routing.  For
+information on how this works, see the 
+[Cloud VPN Overview](https://cloud.google.com/compute/docs/vpn/overview).
+
+There are two ways to create VPN gateways on GCP: using the Google Cloud
+Platform Console and using the
+[gcloud command-line tool](https://cloud.google.com/sdk/).
+This section describes how to perform the tasks using the GCP Console. To see
+the `gcloud` commands for performing these tasks, see the
+[appendix](#appendix-using-gcloud-commands).
+
+### Initial tasks
+
+Complete the following procedures before configuring a static GCP VPN gateway 
+and tunnel.
+
+**Important:** Throughout these procedures, you assign names to entities like
+the VPC network and subnet, IP address, and so on. Each time you assign a name,
+make a note of it, because you often need to use those names in later
+procedures.
+
+#### Select a GCP project name
```

Comment:
Combine these sections:

 #### Create a custom VPC network and subnet
1. [Open the GCP Console](https://console.google.com).
1. At the top of the page, select the GCP project you want to use.

     **Note**: Make sure that you use the same GCP project for all of the GCP
    procedures in this guide.

1. In the GCP Console,
[go to the VPC Networks page](https://pantheon.corp.google.com/networking/networks/list).
1. Click **Create VPC network**.